pythonmod/pythonmod.c

```diff
@@ -252,6 +253,16 @@ log_py_err(void)
 	Py_XDECREF(exc_tb);
 }
 
+/* we only want to unwind Python once at exit */
+void pythonmod_atexit(void)
+{
+   assert(py_mod_count == 0);
+   assert(maimthr != NULL);
+ 
+   PyEval_RestoreThread(mainthr);
+   Py_Finalize();
```

Comment:
- Unbound prefers using `log_assert` instead of plain `assert`;
- This seems like a static function to me;
- Also small typo in `mainthr`.
```suggestion
static void
pythonmod_atexit(void)
{
   log_assert(py_mod_count == 0);
   log_assert(mainthr != NULL);
 
   PyEval_RestoreThread(mainthr);
   Py_Finalize();
```

Reply:
> * Unbound prefers using `log_assert` instead of plain `assert`;

Good to know. Thanks
> * This seems like a static function to me;

Yes. Thanks
> * Also small typo in `mainthr`.

Urgh. Thanks